### PR TITLE
[W5.3][W15-A1] Poh Jie

### PR DESCRIPTION
**Pull request by pohjie**
Not ready for review.

### CODE REVIEW

**Comment by pohjie** [comment]
@okkhoy Sorry for the delay, ready for testing now.

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * A superclass for Phone, Email, and Address. We seek to extract out the common methods.
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java]: not necessary to talk about the sub-classes in the super class header comment, you can do the other way around though

**Comment by okkhoy** [comment]
@pohjie 
Some comments added. Please ack & close the PR after reading comments.

**Comment by pohjie** [comment]
Acknowledged. Thank you.

**Pull request closed without merging**